src/test/java/com/hedvig/notificationService/customerio/web/OnQuoteCreatedEventTest.kt

```diff
@@ -62,50 +63,65 @@ class OnQuoteCreatedEventTest {
     @Test
     fun `does not send event when member signed and event is ordinary`() {
         val quoteCreatedEvent = a.quoteCreatedEvent.build()
-        every { memberService.hasPersonSignedBefore(HasPersonSignedBeforeRequest(SSN, EMAIL)) } returns true
+        every { memberService.hasPersonSignedBefore(HasSignedBeforeRequest(MEMBER_ID, SSN, EMAIL)) } returns true
+        every { memberService.hasRedFlag(MEMBER_ID) } returns false
         eventHandlerToTest.onQuoteCreatedHandleRequest(quoteCreatedEvent, CALL_TIME)
-        verify(inverse = true) {
+        verify(exactly = 0) {
             customerioService.updateCustomerAttributes(
                 MEMBER_ID,
                 any(),
                 CALL_TIME
             )
         }
-        verify(inverse = true) { customerioService.sendEvent(MEMBER_ID, quoteCreatedEvent.toMap()) }
+        verify(exactly = 0) { customerioService.sendEvent(MEMBER_ID, quoteCreatedEvent.toMap()) }
     }
 
     @Test
     fun `does not send event when member has not signed and quote is created from hope`() {
-        every { memberService.hasPersonSignedBefore(HasPersonSignedBeforeRequest(SSN, EMAIL)) } returns false
+        every { memberService.hasPersonSignedBefore(HasSignedBeforeRequest(MEMBER_ID, SSN, EMAIL)) } returns false
+        every { memberService.hasRedFlag(MEMBER_ID) } returns false
         val eventWithQuoteCreatedFromHope = a.quoteCreatedEvent.copy(initiatedFrom = "HOPE").build()
         eventHandlerToTest.onQuoteCreatedHandleRequest(eventWithQuoteCreatedFromHope, CALL_TIME)
-        verify(inverse = true) { customerioService.updateCustomerAttributes(MEMBER_ID, any(), CALL_TIME) }
-        verify(inverse = true) { customerioService.sendEvent(MEMBER_ID, any()) }
+        verify(exactly = 0) { customerioService.updateCustomerAttributes(MEMBER_ID, any(), CALL_TIME) }
+        verify(exactly = 0) { customerioService.sendEvent(MEMBER_ID, any()) }
+    }
+
+    @Test
+    fun `does not send event when member has red flag`() {
+        every { memberService.hasPersonSignedBefore(HasSignedBeforeRequest(MEMBER_ID, SSN, EMAIL)) } returns false
+        every { memberService.hasRedFlag(MEMBER_ID) } returns true
+        val eventWithQuoteCreatedFromHope = a.quoteCreatedEvent.copy(initiatedFrom = "HOPE").build()
+        eventHandlerToTest.onQuoteCreatedHandleRequest(eventWithQuoteCreatedFromHope, CALL_TIME)
+        verify(exactly = 0) { customerioService.updateCustomerAttributes(MEMBER_ID, any(), CALL_TIME) }
+        verify(exactly = 0) { customerioService.sendEvent(MEMBER_ID, any()) }
```

Comment:
In cases like this, I think it's clearer and less error prone to to use any() for all fields like:
```suggestion
        verify(exactly = 0) { customerioService.updateCustomerAttributes(any(), any(), any()) }
        verify(exactly = 0) { customerioService.sendEvent(any(), any()) }
```

Reply:
I agree, good catch. For `exactly = 0` then `any()` is more powerful